lib/loaf/view_extensions.rb

```diff
@@ -68,10 +68,10 @@ def breadcrumb_trail(options = {})
     def current_crumb?(path, pattern = :inclusive)
       return false unless request.get? || request.head?
 
-      origin_path = URI.parser.unescape(path).force_encoding(Encoding::BINARY)
+      origin_path = URI.DEFAULT_PARSER.unescape(path).force_encoding(Encoding::BINARY)
```

Comment:
It seems, looking at the URI source, that `DEFAULT_PARSER` is more of internal implementation detail and we could use public interface `URI.unescape` instead. (see https://github.com/ruby/ruby/blob/0bedb3e5b1e80b5fdac2086f9b0c54df56f7313b/lib/uri/common.rb#L319)